db/db_impl_write.cc

```diff
@@ -106,7 +114,11 @@ Status DBImpl::WriteImpl(const WriteOptions& write_options,
     if (write_thread_.CompleteParallelMemTableWriter(&w)) {
       // we're responsible for exit batch group
       auto last_sequence = w.write_group->last_sequence;
-      versions_->SetLastSequence(last_sequence);
+      if (!concurrent_prepare_ || !disable_memtable) {
```

Comment:
I did assert(!disable_memtable) and this test failed:
```
./transaction_test --gtest_filter=MySQLStyleTransactionTest/MySQLStyleTransactionTest.TransactionStressTest/0
```
I guess it is because we let the followers run independent of disalbe_memtable value.